src/Infrastructure.EntityFramework/Repositories/OrganizationDomainRepository.cs

```diff
@@ -95,6 +95,29 @@ from s in sJoin.DefaultIfEmpty()
         return ssoDetails;
     }
 
+    public async Task<IEnumerable<VerifiedOrganizationDomainSsoDetail>> GetVerifiedOrganizationDomainSsoDetailsAsync(string email)
+    {
+        var domainName = new MailAddress(email).Host;
+
+        using var scope = ServiceScopeFactory.CreateScope();
+        var dbContext = GetDatabaseContext(scope);
+        return await (from o in dbContext.Organizations
+                      from od in o.Domains
+                      join s in dbContext.SsoConfigs on o.Id equals s.OrganizationId into sJoin
+                      from s in sJoin.DefaultIfEmpty()
+                      where od.DomainName == domainName
+                            && o.Enabled
+                            && s.Enabled
+                            && od.VerifiedDate != null
```

Comment:
This is missing some of the conditions from its MSSQL equivalent, like directly checking that the SSO policy is enabled for the organization.

Reply:
Good point. I looked over what I had in the sproc compared to what was being used, and I see that it doesn't actually care about the policy of `SSO Required`. It just cares if SSO is available (if they have a sso config). So, I adjusted the sproc to reflect that, and that matches the EF query.